avocado/core/future/settings.py

```diff
@@ -433,18 +433,16 @@ def merge_with_arguments(self, arg_parse_config):
         :param arg_parse_config: argparse.config dictionary with all
                                  command-line parsed arguments.
         """
-        for tag, value in arg_parse_config.items():
+        for namespace, value in arg_parse_config.items():
             # This check is important! For argparse when an option is
             # not passed will return None. We need to update only the
             # options that the user has specified.
             if value is not None:
                 try:
-                    option = self._namespaces[tag]
+                    _ = self._namespaces[namespace]
```

Comment:
Instead of try.. except here with a non used variable (`_`), maybe we could use `if namespace in self._namespaces`.

Reply:
Agreed... I fell short of doing more changes.